sdk/couchbase-connstr/Cargo.toml

```diff
@@ -6,6 +6,10 @@ edition = "2021"
 [dependencies]
 form_urlencoded = "1.2.1"
 hickory-resolver = { version = "0.24.1", features = ["dns-over-rustls"], default-features = false }
+log = "0.4.22"
 regex = "1.11.0"
 thiserror = "1.0"
 url = "2.5.2"
+
+[dev-dependencies]
+tokio = { version = "1.0", features = ["rt", "net", "macros"] }
```

Comment:
This looks like a very old version of tokio. We seem to have "1.40" and "1.38" in the existing cargo.toml files. I'm not too well informed on the rust dependency management best practices, but this doesn't seem great?

Reply:
Not entirely sure why that was 1.0, it's now 1.40.